### DISCUSSION TITLE
Invalid command 'OIDCPathAuthRequestParams', perhaps misspelled or defined by a module not included in the server configuration

### DISCUSSION THREAD

**Discussion by rarathore** [OpenIDC/mod_auth_openidc#940] [Q&A]
Hi Team,

I tried below configuration in Apache HTTP server

<Location /XYZ/reauth>
	AuthType openid-connect
    <RequireAll>
        Require valid-user
        Require claim acr:1
    </RequireAll>
    OIDCUnAuthAction auth
    OIDCPathAuthRequestParams "prompt=login&acr_values=1"
</Location>

got below error while starting Apache

Invalid command 'OIDCPathAuthRequestParams', perhaps misspelled or defined by a module not included in the server configuration

please let me know what I am missing here, I also added the entry in config file.

LoadModule auth_openidc_module modules/mod_auth_openidc.so

**Comment by zandbelt** [comment]
use the latest version from the Releases page

**Reply by rarathore** [reply]
I am using 2.4.11.3, is it ok?

**Reply by zandbelt** [reply]
yes, then you just have to make sure that the module is loaded, see also `apache2ctl -M`

**Reply by rarathore** [reply]
I checked module is there and if I just comment our below line

OIDCPathAuthRequestParams "prompt=login&acr_values=1"

it is working fine, so no issue with module related settings.
